app/src/test/resources/design-document.md

```diff
@@ -0,0 +1,90 @@
+# Design Document
+
+## _Collector's Stash_ Design
+
+## 1. Problem Statement
+Many individuals have a passion for collecting various and unique items, be it comic books, stamps, or other collectibles. They often resort to carrying physical notepads and books to manually document and keep track of their collections. This traditional method is overwhelming and can also be prone to errors, loss, and inefficiency. To address this issue, I am developing a innovative application aimed at providing collectors with a digital solution for managing and preserving their collections.
```

Comment:
I'd suggesting updating this last sentence to say "comic book collections" now that those are the focus.

Reply:
Changed